files/pull-sandbox-image.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-sandbox_image="$(awk -F'[ ="]+' '$1 == "sandbox_image" { print $2 }' /etc/containerd/config.toml)"
+source <(grep "sandbox_image" /etc/containerd/config.toml | tr -d ' ')
```

Comment:
I though it's a danger more to use "grep" here.

Imagine there's a comment line with "sandbox_image" but it just a comment?

Reply:
TOML comments use a hash symbol (#) like bash, so it "just works". I added an example of this to my testing section in the description.

https://github.com/toml-lang/toml/blob/main/toml.md#user-content-comment